src/validation.ts

```diff
@@ -27,6 +27,10 @@ ajv.addKeyword({
   keyword: 'markdownDescription',
   schemaType: 'string',
 });
+ajv.addKeyword({
+  keyword: 'customOptions',
+  schemaType: 'boolean',
+});
```

Comment:
not needed?
```suggestion
```

Reply:
validation fails otherwise 👀

Reply:
only because the property is not indented correctly..